x-pack/plugins/observability_solution/infra/public/components/logging/log_analysis_results/category_expression.tsx

```diff
@@ -48,16 +47,16 @@ export const RegularExpressionRepresentation: React.FunctionComponent<{
   );
 });
 
-const CategoryPattern = euiStyled.span`
-  font-family: ${(props) => props.theme.eui.euiCodeFontFamily};
+const CategoryPattern = styled.span`
+  font-family: ${(props) => props.theme.euiTheme.font.familyCode};
   word-break: break-all;
 `;
 
-const CategoryPatternWildcard = euiStyled.span`
-  color: ${(props) => props.theme.eui.euiColorMediumShade};
+const CategoryPatternWildcard = styled.span`
+  color: ${(props) => props.theme.euiTheme.colors.mediumShade};
```

Comment:
**nit**: is this colour deprecated and should it change with another one from the palette?

Reply:
There's some comment in #eui slack about deprecated colors, that we can keep for now, let me find it